test/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
```

Comment:
Will more tests go here? As far as I can tell this shell script only tests ubuntu

Reply:
Ya, we'll want to add more `docker exec` calls to this file to setup each of the destinations before we run tests.

I think we'll want to do that in parallel, so maybe this script will be replaced by a Go script when that happens.